packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -236,9 +235,7 @@ export default class ClickableBehavior extends React.Component<Props, State> {
 
     handleKeyUp = (e: SyntheticKeyboardEvent<*>) => {
         const keyCode = e.which || e.keyCode;
-        if (keyCode === keyCodes.tab) {
-            this.setState({focused: true});
-        } else if (
```

Comment:
I removed this because it's not necessary. When tabbing into or away from an element, `onFocus` and `onBlur` respectively fire for that element. Previously, components were updating their state to `focused: false` twice when tabbed away from, once from `handleKeyDown` and once from `handleBlur`. I verified this by adding a temporary logging statement in `componentDidUpdate`.